addons/website_partner/models/res_partner.py

```diff
@@ -8,7 +8,8 @@ class WebsiteResPartner(models.Model):
     _name = 'res.partner'
     _inherit = ['res.partner', 'website.seo.metadata', 'website.published.mixin']
 
-    website_description = fields.Html('Website Partner Full Description', strip_style=True)
+    website_description = fields.Html(
+        'Website Partner Full Description', sanitize=False, groups='website.group_website_publisher')
     website_short_description = fields.Text('Website Partner Short Description')
```

Comment:
@nim-odoo  By removing `strip_style=True` and adding `sanitize=False` forum user can inject css and javascript from Forum profile. It use same field and forum user can update it.

![download 4](https://user-images.githubusercontent.com/7629089/43645410-6e408ee2-974f-11e8-9528-7ad482340883.png)

Reply:
Well spotted, I missed that field!